backend/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to this project will be documented in this file.
 
 ## Unreleased
 
+- Introduced batches in migrations m0037 and logging.
```

Comment:
We had one time a quite long migration file in the past and a partial migration process was developed. Might be worth to check it out in case you were not aware of it. The partial migration was storing its process in the database, so that we could re-start the migration if it fails (or SRE needed to stop/re-start the migration for some reasons). Might be worth to check it it.

https://github.com/Concordium/concordium-scan/blob/main/backend/src/migrations.rs#L361

https://github.com/Concordium/concordium-scan/blob/main/backend/src/migrations/m0019_payday_stake_information.rs